### DISCUSSION TITLE
ui control node usable?

### DISCUSSION THREAD

**Discussion by Sineos** [phoddie/node-red-mcu#89] [Q&A]
I tried to use the node `ui control`. xsbug is telling me:
`Disabling unsupported node type "ui_ui_control"!`
Can it be used?

**Comment by phoddie** [comment]
No, it is not currently implemented.

It looks like some aspects of it do not apply -- the idea of connect & lost seem to be specific to the Node-RED Dashboard implementation that uses the browser for rendering, and so can be disconnected from the Node-RED runtime. On the MCU, they are all a single process.

But other aspects of the node could make sense.

**Comment by Sineos** [comment]
Thanks. Since it reported back this `ui_ui_` (double ui), I was thinking I messed it up.
Just trying to get warm with the NR MCU way and tried to show / hide UI tabs or change them programmatically. Not a biggie
